doc/release-notes-18331.md

```diff
@@ -0,0 +1,8 @@
+Build System
+------------
+
+The source code archives that are provided with gitian builds do not contain
```

Comment:
Would s/do not contain/no longer contain/ to clarify

Reply:
Fixed.